templates/sheets/spell-sheet.html

```diff
@@ -74,9 +86,10 @@ <h1 class="spellname"><input name="name" type="text" value="{{item.name}}" place
                 </select>
             {{/if}}
         </div>
-        <label>{{localize "WITCHER.Item.Effect"}}</label>
-<textarea rows="10" name="data.effect">{{data.effect}}</textarea>
+    </div>
 
+    <label>{{localize "WITCHER.Item.Effect"}}</label>
+    <textarea rows="10" name="data.effect">{{data.effect}}</textarea>
```

Comment:
There is a problem with the textare that adds spaces when modified.  
You need to leave no spaces in front of it to prevent this.

Reply:
Okey weird bug :P But I will fix it.